examples/simpletest.py

```diff
@@ -0,0 +1,13 @@
+import time
+import board
+import adafruit_hcsr04
+
+sonar = adafruit_hcsr04.HCSR04(trigger_pin=board.D5, echo_pin=board.D6)
+
+while True:
+    try:
+        print((sonar.distance,))
```

Comment:
Why is this in a tuple? `print(sonar.distance)` is simpler.

Reply:
so you can use the plotter :)

Reply:
Ah! Ok